packages/cicero-core/test/templatelibrary.js

```diff
@@ -14,9 +14,11 @@
 
 'use strict';
 
-const TemplateLibrary = require('../lib/templatelibrary');
+let TemplateLibrary = require('../lib/templatelibrary');
```

Comment:
That doesn't look right...

Reply:
@mttrbrts I think I made this mistake.Revert this one.

Reply:
This is correct with `let`. I reload the dependency in a test to bust the module cache so that I can add the mock